ci-operator/config/openstack-k8s-operators/nova-operator/openstack-k8s-operators-nova-operator-master.yaml

```diff
@@ -46,7 +46,7 @@ tests:
 - as: functional
   commands: |
     export GOFLAGS=
-    make gotest
+    make test GINKGO_ARGS='--no-color'
```

Comment:
ya i would prefer to keep them locally so I'm glad we can do this via an env var in the ci job